src/NUnitFramework/framework/Internal/ParamNumberConversions.cs

```diff
@@ -0,0 +1,78 @@
+using System;
```

Comment:
Could you add the NUnit copyright notice to this new file, and your new test file please. 😄